src/components/SavingMessage.css

```diff
@@ -0,0 +1,59 @@
+.blob-wrapper {
```

Comment:
Should we move this CSS code to the Style Guide?

Reply:
The wrapper is better manage in the implementation and the animation maybe in the library @GusBaamonde what do you think about this?